### PR TITLE
Cleanup, additional linting, and build badge

### PR DESCRIPTION
**Pull request by jsf9k**
@dav3r, @felddy, and @KyleEvers, please have a look at this PR.  I've been toying with the idea of running linters in TravisCI on every file I can in each repo.  In this repo I added a Markdown linter, and I wanted to know what you guys thought of doing this in all our repos.  I'm for it, if only to ensure that we standardize.  In cases where you really need to, you can always forcibly disable certain warnings.  In this case, for example, I took the particularly heavy-handed step of  not running the Markdown linter on [LICENSE.md](/cisagov/ansible-role-banner/blob/develop/LICENSE.md) at all, since that file needs to match the license text exactlyin order got GitHub's automatic recognition fu to work.  Other than that, I only had to make a handful of other changes in order to make the linter happy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,4 +43,4 @@ with this waiver of copyright interest.
 
 ## Author Information ##
 
-Mark Feldhousen <make.feldhousen@beta.dhs.gov>
+Mark Feldhousen <mark.feldhousen@trio.dhs.gov>
```
Comment (dav3r) [inline README.md]: LOL - "make.feldhousen" 😃

**Review by dav3r — APPROVED** [review]
I'm on board with trying to lint all the things- your plan makes sense to me.  We can always change course later if we encounter any problems with this approach.

File: README.md (hunk 1 above)
Comment (felddy) [reply README.md]: I made three Feldhousen.

**Comment by felddy** [comment]
I am for linting all the things.  I'd rather a cold lump of metal attack your ego than doing it myself.

**Comment by jsf9k** [comment]
Based on @dav3r's and @felddy's comments I'll go full Cajun on this repo and blacken it.

Blackened Redfish

**Comment by jsf9k** [comment]
@cisagov/team-ois, please have a look at the additions I made to `CONTRIBUTING.md` and let me know what you think.  Once that is good I will use this as a basis for the bootstrap repo we talked about.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: :baby: :baby: :baby:

**Pull request merged**